okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -156,6 +158,17 @@ public final OkHttpChannelBuilder transportExecutor(@Nullable Executor transport
     return this;
   }
 
+  /**
+   * Override the default {@link SocketFactory} used to create sockets. If the socket factory is not
+   * set, the value of {@link SocketFactory#getDefault} will be used.
```

Comment:
Let's hide the implementation detail. What about "If the socket factory is not set or set to null, a default one will be used"?

Reply:
Done